stable/yugabyte/templates/service.yaml

```diff
@@ -386,7 +386,7 @@ spec:
           {{- if $rpcPreflight -}}{{ $rpcPreflight | nindent 12 }}{{ end -}}
           {{- $broadcastAddr := include "yugabyte.server_broadcast_address" $serviceValues -}}
           {{/* skip bind check for Istio multi-cluster, we cannot/don't bind to service IP */}}
-          {{- if (not (and $root.Values.istioCompatibility.enabled $root.Values.multicluster.createServicePerPod)) }}
+          {{- if (not (or $root.Values.istioCompatibility.enabled $root.Values.multicluster.createServicePerPod)) }}
```

Comment:
We can modify the comment as we are decoupling servicePerPod and istioCompatibility. 
```
{{/* skip bind check for servicePerPod multi-cluster, we cannot/don't bind to service IP */}}
```
Also, this skip thing is specific to servicePerPod so if can be just:
```
{{- if not $root.Values.multicluster.createServicePerPod }}
```

Reply:
done